Gemfile.lock

```diff
@@ -1,256 +0,0 @@
-GEM
```

Comment:
Why did we remove Gemfile.lock?

Reply:
My mistake, I believed that is not necessary. I push it again.